### PR TITLE
add section to zar README describing graph queries

### PR DESCRIPTION
**Pull request by mccanne**
This commit adds some color to the zar README for how you can do graph queries with zar.

### CODE REVIEW

File: cmd/zar/README.md [hunk 1]
```diff
@@ -406,6 +406,74 @@ URI                     SUM
...
+This command sequence will collect up the edges into `edges.njdson`:
+```
+zar find -z -x graph 216.58.193.195 | zq "count() by from,to" - > edges.zng
+zar find -z -x graph 10.47.6.162 | zq "count() by from,to" - >> edges.zng
+zar find -z -x graph 10.47.5.153 | zq "count() by from,to" - >> edges.zng
+zq -f ndjson "sum(count) as value by from,to | put source=from, target=to | cut source,target,value" edges.zng >> edges.ndjson
+```
+> (Note: with a few additions to zql and zar, we can make this much simpler and
+> more efficient.  Coming soon.  Also, we should be able to say `group by node`,
+> which implies no reducer and emits columns with just the group-by keys.)
```
Comment (henridf) [inline cmd/zar/README.md:464]: (Supporting just `group by node` should just be small grammar change, btw... the AST and runtime already support it).

File: cmd/zar/README.md [hunk 2]
```diff
@@ -406,6 +406,74 @@ URI                     SUM
...
+could prove useful for your archive analytics.
+
+For example, to build an searchable edge list comprising communicating IP addresses,
+run the following commands to a create a multi-key search index
+keyed by all unique instances of IP address pairs (in both directions)
+where each index includes a count of the occurrences.
+```
+zar zq -o forward.zng "id.orig_h != null | put from=id.orig_h,to=id.resp_h | count() by from,to" _
+zar zq -o reverse.zng "id.orig_h != null | put from=id.resp_h,to=id.orig_h | count() by from,to" _
+zar zq -o directed-pairs.zng "count() by from,to" forward.zng reverse.zng
```
Comment (alfred-landrum) [inline cmd/zar/README.md:426]:
```suggestion
zar zq -o directed-pairs.zng "sum(count) as count by from,to" forward.zng reverse.zng
```

File: cmd/zar/README.md [hunk 3]
```diff
@@ -406,6 +406,74 @@ URI                     SUM
...
+zar zdx -o graph -k from,to directed-pairs.zng
+```
+> (Note: there is a small change we can make to zql to do this with one `zar zq`
+> command... coming soon.)
+
+This creates an index called "graph" that you can use to search for IP address
+pair relationships, e.g., you can say
+```
+zar find -z -x graph 216.58.193.195 | zq -f table "count() by from,to" -
```
Comment (alfred-landrum) [inline cmd/zar/README.md:435]:
```suggestion
zar find -z -x graph 216.58.193.195 | zq -f table "sum(count) as count by from,to" -
```

File: cmd/zar/README.md (hunk 2 above)
Comment (alfred-landrum) [inline cmd/zar/README.md:426]:
```suggestion
zar zq -o directed-pairs.zng "sum(count) as count by from,to" forward.zng reverse.zng
```

File: cmd/zar/README.md (hunk 1 above)
Comment (henridf) [reply cmd/zar/README.md:464]: https://github.com/brimsec/zq/issues/821

Comment (mccanne) [reply cmd/zar/README.md:464]: cool thanks

File: cmd/zar/README.md (hunk 3 above)
Comment (mccanne) [reply cmd/zar/README.md:435]: oops... I'll fix the example table output too.  I should have realized something was wrong when the counts were too small!  thx

**Pull request merged**